vendor/github.com/openshift/hive/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -206,6 +206,10 @@ type Provisioning struct {
 	// Defaults to openshift-install if none specified.
 	// +optional
 	InstallStrategy *InstallStrategy `json:"installStrategy,omitempty"`
+
+	// ServiceAccountIssuerKeySecretRef refers to a Secret that contains a 'bound-service-account-signing-key.key' data key pointing to the private key that will be used to sign ServiceAccount objects. Primarily used to provision AWS clusters to use Amazon's Security Token Service.
+	// +optional
+	ServiceAccountIssuerKeySecretRef *corev1.LocalObjectReference `json:"serviceAccountIssuerKeySecretRef,omitempty"`
```

Comment:
Would this be better named BoundServiceAccountSigningKeySecretRef to match the installer filename?

Also why am I not seeing the change in top level apis/ ? Only the vendored version?

Reply:
Lol I've been ctrl-p'ing in Vim, picking up the vendored copy, modifying it, everything works, and missing the actual source code in apis. Gonna have to watch out for that.